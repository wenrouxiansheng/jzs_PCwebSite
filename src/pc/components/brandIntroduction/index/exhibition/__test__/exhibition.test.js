import React from "react";
import renderer from "react-test-renderer";
import Exhibition from "../index";

describe("PC:<Exhibition />", () => {
  it("render", () => {
    const wrapper = renderer.create(<Exhibition />);
    let tree = wrapper.toJSON();

    //渲染测试
    expect(tree).toMatchSnapshot();
  });
});
