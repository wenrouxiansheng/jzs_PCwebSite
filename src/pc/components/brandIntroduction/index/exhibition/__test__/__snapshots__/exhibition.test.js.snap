// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PC:<Exhibition /> render 1`] = `
<div
  className="exhibition"
  style={Object {}}
>
  <div
    className="fontContainer"
  >
    <div>
      <h3>
        中国艺术教育知名品牌  实现对自我的艺术追求
      </h3>
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "桔子树艺术教育，是中国艺术教育培训知名品牌。成立逾9年，总部位于北京市通州区万达广场A座，是集青少儿才艺培训、成人艺术培训、艺术课程研发、艺考辅导、企业服务、参赛培训等为一体的艺术教育培训综合体。",
          }
        }
      />
    </div>
    <strong>
      01
    </strong>
  </div>
  <div
    className="imgContainer"
  >
    <div>
      <img
        alt=""
      />
    </div>
    <strong>
      02
    </strong>
  </div>
</div>
`;
